### ISSUE TITLE
[Wrath Bug Report] No cast bar for channeled spells

### ISSUE THREAD

**Issue by Robert86** [tukui-org/ElvUI#707]
### What is the issue you are having?

Cast bar does not show while channeling spells. Cast bar works fine for other spells. Only channeled spells that has the issue.
It works in version 13.10 but not 13.13. Havn't tried the other versions between.
I have ticks enabled but didn't try without ticks enabled in 13.13.

### What is the expected behavior?

Cast bar should show.

### What actually happened?

Cast bar didn't show and got a lua error.

### Errors

```shell
3x ...s\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:525: attempt to index field 'ChainChannelTicks' (a nil value)
[string "@Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua"]:525: in function `PostCastStart'
[string "@Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\castbar.lua"]:324: in function <...AddOns\ElvUI_Libraries\Core\oUF\elements\castbar.lua:215>
[string "=(tail call)"]: ?

Locals:
self = ElvUF_Player_CastBar {
 0 = <userdata>
 ModuleStatusBars = <table> {
 }
 CustomDelayText = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:370
 stagePoints = <table> {
 }
 CustomTimeText = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:400
 bg = <unnamed> {
 }
 spellName = "Hellfire"
 unit = "player"
 ButtonIcon = <unnamed> {
 }
 duration = 12.589000
 tickWidth = 2
 UpdatePipStep = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:31
 holdTime = 0
 startTime = 55409.536000
 max = 12.589000
 empowering = false
 Pips = <table> {
 }
 pipColor = <table> {
 }
 CreatePip = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:100
 Spark_ = <unnamed> {
 }
 spellID = 47823
 channeling = true
 tickHeight = 22
 LatencyTexture = <unnamed> {
 }
 Time = <unnamed> {
 }
 isTransparent = false
 tickColor = <table> {
 }
 hookedColor = true
 backdropTex = <unnamed> {
 }
 SetStatusBarColor = <function> defined =[C]:-1
 invertColors = false
 custom_backdrop = false
 Icon = <unnamed> {
 }
 PostUpdatePip = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:41
 __owner = ElvUF_Player {
 }
 Holder = <unnamed> {
 }
 notInterruptible = false
 SafeZone = <unnamed> {
 }
 PostCastStop = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:593
 Text = <unnamed> {
 }
 Spark = <unnamed> {
 }
 PostCastStart = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:491
 ForceUpdate = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\castbar.lua:567
 PostCastInterruptible = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:611
 PostCastFail = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\Elements\CastBar.lua:600
 PixelSnapDisabled = true
 timeToHold = 0
 casting = false
 delay = 0
 backdrop = <unnamed> {
 }
}
unit = "player"
parent = ElvUF_Player {
 0 = <userdata>
 UNIT_COMBAT = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\combatindicator.lua:59
 SetHeight = <function> defined =[C]:-1
 ResurrectIndicator = <unnamed> {
 }
 UNIT_SPELLCAST_CHANNEL_START = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\castbar.lua:215
 SetAlpha = <function> defined =[C]:-1
 unitEvents = <table> {
 }
 AuraHighlightBackdrop = false
 USE_INSET_POWERBAR = false
 UNIT_MAXHEALTH = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\ouf.lua:1097
 __customargs = <table> {
 }
 TargetGlow = <unnamed> {
 }
 PowerPrediction = <table> {
 }
 Portrait3D = <unnamed> {
 }
 AuraHighlight = <unnamed> {
 }
 USE_POWERBAR = false
 PLAYER_REGEN_DISABLED = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\combatindicator.lua:59
 RaisedElementParent = <unnamed> {
 }
 UNIT_SPELLCAST_CHANNEL_UPDATE = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\castbar.lua:330
 PvPIndicator = <unnamed> {
 }
 INFO_PANEL_HEIGHT = 0
 FadeObject = <table> {
 }
 UNIT_EXITED_VEHICLE = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\ouf.lua:73
 __mousetags = <table> {
 }
 UNIT_CONNECTION = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\health.lua:143
 Debuffs = ElvUF_PlayerDebuffs {
 }
 PARTY_LOOT_METHOD_CHANGED = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\masterlooterindicator.lua:82
 UNIT_HEAL_PREDICTION = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\ouf.lua:1097
 PixelSnapDisabled = true
 CLASSBAR_DETACHED = true
 Update = <function> defined @Interface\AddOns\ElvUI\Core\Modules\UnitFrames\UnitFrames.lua:1048
 UNIT_SPELLCAST_SENT = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\castbar.lua:115
 UNIT_SPELLCAST_DELAYED = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\oUF\elements\castbar.lua:330
 USE_MINI_POWERBAR = false
 EnergyManaRegen = <unnamed> {
 }
 PARTY_MEMB
```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://www.tukui.org/download.php?ui=elvui&changelog).
- [X] I have verfied ElvUI is up to date by checking /estatus.
- [X] I have verfied my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by FobbanN12** [comment]
I am experiencing the same issue since version 13.13. It was not present in version 13.11

**Comment by Azilroka** [comment]
It has been fixed in dev

**Comment by kodewdle** [comment]
fixed in 13.14 my bad guys <3 🌼
